pkg/status/README.md

```diff
@@ -0,0 +1,12 @@
+# pkg/status
+
+## Overview
+
+This module contains functionality for aggregating status information about components. Used by extensions that
+implement the `extension.StatusWatcher` interface for collecting status information about the internal status of all
```

Comment:
```suggestion
implement the `componentstatus.Watcher` interface for collecting status information about the internal status of all
```

The interface was renamed on core. I suspect there are some stale references in docs / comments that need to be updated.